code/lib/core-common/src/utils/__tests__/normalize-stories.test.ts

```diff
@@ -230,7 +230,7 @@ describe('normalizeStoriesEntry', () => {
       {
         "titlePrefix": "",
         "directory": ".",
-        "files": "**/*.@(mdx|stories.mdx|stories.tsx|stories.ts|stories.jsx|stories.js)",
+        "files": "**/*.@(mdx|stories.@(tsx|ts|jsx|js))",
```

Comment:
Whoa. Is this a valid glob?

Reply:
It seems to work! So I guess so.

Reply:
We've been using it for a while in our sandboxes: https://github.com/storybookjs/storybook/blob/7f9e995af04c11e570cc5f5e18737491fff60dcb/scripts/tasks/sandbox-parts.ts#L258